{"uuid":"79e96d84-dba3-4eb8-8fb9-c9f66bd71d3a","name":"Scenario: User enters Invalid Python code and Presented with Error Message","children":["7aa894a1-b3bd-4390-be3e-7f7596a4440b"],"befores":[],"afters":[{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681568393082,"stop":1681568393083},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681568393799,"stop":1681568393800},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681568393963,"stop":1681568393964},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681568394089,"stop":1681568394090},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681568394114,"stop":1681568394115}],"start":1681568392868,"stop":1681568394115}